### ISSUE TITLE
the MAP of verification suddenly becomes 0 during training

### ISSUE THREAD

**Issue by yrn1126** [ultralytics/yolov5#8339]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

![微信截图_20220625184517](https://user-images.githubusercontent.com/102804183/175770615-3eb41512-a0a2-493d-b885-96fb1d49f246.png)
![微信截图_20220625184551](https://user-images.githubusercontent.com/102804183/175770620-74884471-9a10-4a9a-8281-84fe0d13b186.png)
During the training of my own dataset, everything was normal. But at the 80th epoch, the mAP of the verification became 0 suddenly . After 10 epochs, it becomes normal again, but the mAP was much lower than before. I feel confused about this question.

**Comment by josebenitezg** [comment]
Hi @yrn1126 I have the same problem

**Comment by glenn-jocher** [comment]
@josebenitezg @yrn1126 your losses are increasing.

**Comment by josebenitezg** [comment]
hi @glenn-jocher 
Actually, first are decreasing but then are incresing again, and recover again.
I try with yolov5s6 model with same dataset and I don't have this problem.
<img width="367" alt="Captura de Pantalla 2022-06-26 a la(s) 12 51 51" src="https://user-images.githubusercontent.com/58047828/175822817-fdaf0237-e5e3-4fdf-a123-f806b9839421.png">
<img width="1019" alt="Captura de Pantalla 2022-06-26 a la(s) 12 47 39" src="https://user-images.githubusercontent.com/58047828/175822821-17641ceb-c95a-42b1-a306-b6616cfae933.png">

**Comment by glenn-jocher** [comment]
@josebenitezg you may need to use less aggressive hyps or a longer warmup etc. to increase stability.

**Comment by josebenitezg** [comment]
@glenn-jocher  I put the default hyps but still same error  with large model (even with the last commit)
As you can see here, I train yolov5s6 with no problem (orange one). Same dataset with large suddenly becomes 0

`hyperparameters: lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=0.05, cls=0.5, cls_pw=1.0, obj=1.0, obj_pw=1.0, iou_t=0.2, anchor_t=4.0, fl_gamma=0.0, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0`
Training on EC2 Instance p3.2xlarge Instance
<img width="710" alt="Captura de Pantalla 2022-06-27 a la(s) 13 13 23" src="https://user-images.githubusercontent.com/58047828/175987201-2d43585f-d8b6-41b8-896a-cbe6eddc1e42.png">
<img width="713" alt="Captura de Pantalla 2022-06-27 a la(s) 13 13 15" src="https://user-images.githubusercontent.com/58047828/175987204-fd70759a-2e7c-4154-af43-a4c8d2f66153.png">

**Comment by glenn-jocher** [comment]
@josebenitezg perhaps related to warmup bug fix https://github.com/ultralytics/yolov5/pull/8356 merged today. You should git pull and train again.
